x-pack/plugins/security_solution/public/timelines/components/timeline/unified_components/data_table/index.tsx

```diff
@@ -0,0 +1,495 @@
+
+      return {
+        columns: columnSettings,
+      };
+    }, [columns]);
+
+    const defaultColumnIds = useMemo(() => columns.map((c) => c.id), [columns]);
+
+    const { timeline: { filterManager, rowHeight, sampleSize } = timelineDefaults } = useSelector(
+      (state: State) => timelineBodySelector(state, timelineId)
+    );
+
+    // const { activeStep, isTourShown, incrementStep } = useTourContext();
+
+    /* const isTourAnchor = useMemo(
+        () =>
+          isTourShown(SecurityStepId.alertsCases) &&
+          eventType === 'signal' &&
+          isDetectionsAlertsTable(timelineId) &&
+          ariaRowindex === 1,
+        [isTourShown, ariaRowindex, eventType, timelineId]
+      );
+      const onExpandEvent = useCallback(() => {
+        if (
+          isTourAnchor &&
+          activeStep === AlertsCasesTourSteps.expandEvent &&
+          isTourShown(SecurityStepId.alertsCases)
+        ) {
+          incrementStep(SecurityStepId.alertsCases);
+        }
+        onEventDetailsPanelOpened();
+      }, [activeStep, incrementStep, isTourAnchor, isTourShown, onEventDetailsPanelOpened]);*/
+
+    /* const onRowSelected: OnRowSelected = useCallback(
+        ({ eventIds, isSelected }: { eventIds: string[]; isSelected: boolean }) => {
+          dispatch(
+            timelineActions.setSelected({
+              id,
+              eventIds: getEventIdToDataMapping(data, eventIds, queryFields),
+              isSelected,
+              isSelectAllChecked:
+                isSelected && Object.keys(selectedEventIds).length + 1 === data.length,
+            })
+          );
+        },
+        [data, dispatch, id, queryFields, selectedEventIds]
+      );*/
+
+    const tableRows: Array<DataTableRecord & TimelineItem> = useMemo(
+      () =>
+        events.map(({ _id, _index, ecs, data }) => {
+          const _source = ecs as unknown as Record<string, unknown>;
+          const hit = { _id, _index: String(_index), _source };
+          return {
+            _id,
+            id: _id,
+            data,
+            ecs,
+            raw: hit,
+            flattened: flattenHit(hit, dataView, {
+              includeIgnoredValues: true,
+            }),
+          };
+        }),
+      [events, dataView]
+    );
+
+    const handleOnEventDetailPanelOpened = useCallback(
+      (eventData: DataTableRecord & TimelineItem) => {
+        const updatedExpandedDetail: ExpandedDetailType = {
+          panelView: 'eventDetail',
+          params: {
+            eventId: eventData.id,
+            indexName: eventData._index ?? '', // TODO: fix type error
+            refetch,
+          },
+        };
+
+        dispatch(
+          timelineActions.toggleDetailPanel({
+            ...updatedExpandedDetail,
+            tabType: TimelineTabs.query,
+            id: timelineId,
+          })
+        );
+
+        activeTimeline.toggleExpandedDetail({ ...updatedExpandedDetail });
```

Comment:
@PhilippeOberti can we just drop this now. I'm pretty sure you removed it from timeline already?

Reply:
hmmm, the [PR](https://github.com/elastic/kibana/pull/177087) hasn't been merged yet, plus the expandable flyout will be behind a feature flag for a release or so, so let's maybe not remove this code yet?